app/views/welcome/_footer.html.erb

```diff
@@ -0,0 +1,26 @@
+<footer class="page-footer grey darken-4">
+  <div class="container">
+    <div class="row">
+      <div class="col l6 s12">
+        <h5 class="white-text">Footer Content</h5>
+        <p class="grey-text text-lighten-4">This is where the motto of finden will be</p>
+        <%= link_to image_tag('mh_logo.png', width: 100, height: 100), class: 'brand-logo'%>
```

Comment:
Please add a space after `'brand-logo'`. Remember that every ERB tag has to have space at the beginning and the end. There is several code with this issue, please check all your changes before uploading them.